pylint/checkers/base/name_checker/checker.py

```diff
@@ -164,6 +164,12 @@ class NameChecker(_BasicChecker):
             'Emitted when both the "covariant" and "contravariant" '
             'keyword arguments are set to "True" in a TypeVar.',
         ),
+        "C0131": (
```

Comment:
Shit, I didn't pay enough attention. We already emit W0130.

C0131 is fine though.

Could you perhaps renumber these? Giving `C0130` to `typevar-double-variance`? We haven't released this yet so we should be good to do so!

Reply:
Ahh it's because W130 (`duplicate-values`) was added since I made the original PR

Will rename `typevar-double-variance` to `C0132` 👍

Reply:
I ended up changing `typevar-double-variance` to `C0131` and `typevar-name-mismatch` to `C0132`

Reply:
Thanks a lot!